content/events/2023-10-18.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 title: Future Talk
-date: 2022-10-18
+date: 2023-10-18
 description: Placeholder description
 location: 1595 Wynkoop St, Denver, CO 80202
 speakers:
-  - name: John Doe
+  - name: Bill Gates, Linus Torvalds, Snoop Dog
```

Comment:
Now that's a talk I would go to. Bill Gates and Linus Torvalds would be very interesting talks. And to top it off with Snoop? I would love that to be a panel discussion about the future in tech with Snoop just sitting back saying yeas to everything with a fun aniocodate.

Reply:
Updates made - thanks. You'd be in good company presenting with titans of industry.